docs/spec/v3.rst

```diff
@@ -1,4 +1,4 @@
-.. _spec_v2:
+.. _spec_v3:
```

Comment:
This change looks ok to me

cc @joshmoore (in case you have thoughts)

Reply:
Sounds like this is good

xref: https://github.com/zarr-developers/zarr-python/issues/1181#issuecomment-1277444650